src/langcoder.cc

```diff
@@ -275,14 +275,18 @@ quint32 LangCoder::guessId( const QString & lang )
 
 std::pair< quint32, quint32 > LangCoder::findLangIdPairFromName( QString const & name )
 {
-  static QRegularExpression reg( "(?=([a-z]{2,3})-([a-z]{2,3}))", QRegularExpression::CaseInsensitiveOption );
+  static QRegularExpression reg( "(^|[^a-z])((?<lang1>[a-z]{2,3})-(?<lang2>[a-z]{2,3}))($|[^a-z])",
```

Comment:
`(^|[^a-z])` seems weird, but regex cannot put the `^ -> beginning of str` inside `[]` .